mod/toggle_language/views/default/toggle_language/toggle_lang.php

```diff
@@ -14,6 +14,11 @@
 		}
 
 		$cookie_name = "lang";
+
+		if (isset($SESSION['language']) ){
```

Comment:
do you mean !isset(), or does the bug only pop up when language is not English?

Reply:
sorry, wrong branch